packages/shopping/src/__tests__/acceptance/user-order.controller.acceptance.ts

```diff
@@ -35,8 +42,11 @@ describe('UserOrderController acceptance tests', () => {
     const userId = user.id;
     const order = givenAOrder({userId: userId, orderId: '1'});
 
+    const token = await authenticateUser(user);
```

Comment:
I think we can just generate the token once, then use it in all the `it` tests.

Reply:
actually every test iteration calls givenAUser() to create a new user for every test. so a token has to be created every time for the new user.